Makefile

```diff
@@ -82,6 +82,7 @@ run: manifests generate fmt vet ## Run a controller from your host.
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
 	docker build -t ${IMG} .
+	docker image prune -f --filter label=stage=mctc-builder
```

Comment:
does this mean that image would need to be rebuild each time? Doesn't that take a while? Would it be better if we could not prune the latest one?

Reply:
`make docker-build` is only really used for local development, images will be built elsewhere for latest tags etc.. What is the actual size of teh images being produced because they really shouldn't be that big

Reply:
Not sure why we need to remove images. Are you saying it produces multiple builder images?

Reply:
@philbrookes It would only delete the builder image that is not used. That would be rebuilt anyways if there's a change in the code

@mikenairn Yes. The reason is that when testing changes that require to re-build the image it's easy to end up with lots of those intermediate images taking space in the computer

Reply:
yeah ok, I don't see why you'd ever rebuild if you'd not changed the code.